### PR TITLE
update notebook base tag

### LINKED ISSUE #83: Update to use Jupyter Notebook 4.1
* [x] bump base image to `jupyter/minimal-notebook:761010eda90d`
* [x] update scala spark kernel build steps
* [ ] get resolution to SciRuby/sciruby#45 (can't install sciruby)

### PR DESCRIPTION
**Pull request by minrk**
closes #83

### CODE REVIEW

**Comment by minrk** [comment]
Running into the sciruby issues in #83. Currently tempted to remove ruby from the demo image, but I'll see if pinning it to the last working version is enough.

**Comment by minrk** [comment]
Pinning mixed_models to 0.1.0 works

**Comment by minrk** [comment]
Spoke too soon, no it doesn't. I'm going to remove ruby from the demo image for now, so we can ship the update.

**Comment by minrk** [comment]
I'm also going to have to pin the spark-kernel to the commit at the time of last build, since it doesn't build anymore. I'm not going to update to toree in this PR, but we should do that separately.

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # Docker demo image, as used on try.jupyter.org and tmpnb.org
 
-FROM jupyter/minimal-notebook:4.0
+FROM jupyter/minimal-notebook:2d878db5cbff
```
Comment (parente) [inline Dockerfile]: We should probably go to a much newer version of the stacks if we're just going to skip ruby and move for now:

https://github.com/jupyter/docker-stacks/wiki/Docker-build-history

Latest there is 4.1. The Notebook 4.2 PR is still pending on stacks waiting for ipywidgets in conda. (Haven't checked in a few days. Will in a minute.)

Comment (parente) [inline Dockerfile]: Bah. Nevermind. Need more coffee. That is the latest.

**Pull request merged**